Source/WebKit/UIProcess/glib/WebProcessPoolGLib.cpp

```diff
@@ -133,6 +133,8 @@ void WebProcessPool::platformInitializeWebProcess(const WebProcessProxy& process
         parameters.hostClientFileDescriptor = UnixFileDescriptor { wpe_renderer_host_create_client(), UnixFileDescriptor::Adopt };
         parameters.implementationLibraryName = FileSystem::fileSystemRepresentation(String::fromLatin1(wpe_loader_get_loaded_implementation_library_name()));
     }
+
+    parameters.availableInputTypes = availableInputTypes();
```

Comment:
In addition to the initial parameter we need an IPC message to be sent to all web processes when the capabilities change.

Reply:
I've created an `UpdateAvailableInputDevices` message, let me know what you think